src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -126,6 +128,43 @@
                                  :else elem))
                              (:data props))}])])})
 
+(react/defc OperationDialog
+            {:render
+             (fn [{:keys [props state]}]
+                 [comps/OKCancelForm
+                  {:header "Operation Details"
+                   :content
+                   (let [server-response (:server-response @state)]
+                        (cond
+                         (nil? server-response)
+                         [:div {} [comps/Spinner {:text "Loading operation details..."}]]
+                         (not (:success? server-response))
+                         (style/create-server-error-message (:response server-response))
+                         :else
+                         [CodeMirror {:text (:raw-response server-response)}]))
+                   :show-cancel? false
+                   :ok-button {:text "Done" :onClick modal/pop-modal}}])
+
+             :component-did-mount
+             (fn [{:keys [props state]}]
+                 (endpoints/call-ajax-orch
+                  {:endpoint
+                   (endpoints/get-workspace-genomic-operations
+                    ;; strip out operations/ from the job id
+                    (last (string/split (:job-id props) #"/")))
+                   :on-done (fn [{:keys [success? get-parsed-response status-text raw-response]}]
+                                (swap! state assoc :server-response
+                                       {:success? success?
+                                        :response (if success? (get-parsed-response false) status-text)
+                                        :raw-response raw-response}))}))})
+
+(react/defc OperationLink
+            {:render
+             (fn [{:keys [props]}]
+                 (let [{:keys [job-id]} props]
+                      [:a {:href "javascript:;"
+                           :onClick #(modal/push-modal [OperationDialog props])} job-id]))})
```

Comment:
In general, it's nice to avoid creating React components if possible, for performance reasons.  This doesn't have any state, so it can just be a function:
```
(defn operation-link [job-id]
  [:a ... ])
```
Also, we like to use `style/create-link` unless we specifically want the default "download blue" and underline, typically for external links.